contracts/src/main/kotlin/com/r3/developers/apples/states/AppleStamp.kt

```diff
@@ -0,0 +1,19 @@
+package com.r3.developers.apples.states
+
+import com.r3.developers.apples.contracts.AppleStampContract
+import net.corda.v5.ledger.utxo.BelongsToContract
+import net.corda.v5.ledger.utxo.ContractState
+import java.security.PublicKey
+import java.util.*
+
+@BelongsToContract(AppleStampContract::class)
+class AppleStamp(
```

Comment:
```suggestion
class AppleStampState(
```

Reply:
This is an interesting discussion.

I asked in the masterclass today with @MrMatthewLayton about the naming convention for States. Whilst it is not set in stone, one school of thought is not to explicitly say State for the states.

For now:
- like most of my responses to the other comments, I will follow the same practice/standard the Apples tutorial gives me so that people who follow the guides can see line-by-line the exact same thing expected of them.
- in this case: [the code in the guide](https://docs.r3.com/en/platform/corda/5.0/developing-applications/basic-cordapp/state.html#add-imports) is what I'll follow

For the future:
- I will make another PR to update the apples tutorial solution to be written in the best practices we decide to go forward with. Yes, we should agree together what we wish to _encourage_ developers to use

Reply:
captured in https://r3-cev.atlassian.net/browse/CORE-15314 and https://r3-cev.atlassian.net/browse/CORE-15315

one to do in a future PR.

resolving